packages/frontend/src/features/common/type.ts

```diff
@@ -0,0 +1,9 @@
+export type UserProps = {
```

Comment:
[must]
自分で提示しておきながらで申し訳ないのですが、これはpropsでは無いので、Userがいいと思います
```suggestion
export type User = {
```